packages/ripple/package.json

```diff
@@ -22,9 +22,6 @@
   "dependencies": {
     "@leafygreen-ui/tokens": "^1.4.0"
   },
-  "devDependencies": {
```

Comment:
confused how we don't need this as a dev dep?

Reply:
Since none of the .story or .spec files explicitly import `lib`, we can rely on the global installation of `react` for development purposes